observability/otel/otel-updating.adoc

```diff
@@ -10,6 +10,8 @@ For version upgrades, the {OTELOperator} uses the Operator Lifecycle Manager (OL
 
 The OLM runs in the {product-title} by default. The OLM queries for available Operators as well as upgrades for installed Operators.
 
+The {OTELOperator} automatically upgrades all `OpenTelemetryCollector` custom resources during its startup. The Operator reconciles all managed instances during its startup: if there is an error, the Operator retries the upgrade at exponential backoff. If an upgrade fails, the Operator will retry the upgrade again when it restarts.
```

Comment:
```suggestion
The {OTELOperator} automatically upgrades all `OpenTelemetryCollector` custom resources during its startup. The Operator reconciles all managed instances during its startup. If there is an error, the Operator retries the upgrade at exponential backoff. If an upgrade fails, the Operator retries the upgrade again when it restarts.
```

Reply:
Thank you. I split the sentence as you suggested, but I kept the future tense as an exception because the future action will take place at a later time.